tests/integration/cli/test_export_inference.py

```diff
@@ -20,8 +20,8 @@ def _check_relative_metric_diff(ref: float, value: float, eps: float) -> None:
 
     avg = max(0.5 * (ref + value), 1e-9)
     diff = abs(value - ref)
-
-    assert diff / avg <= eps, f"Relative difference exceeded {eps} threshold. Absolute difference: {diff}"
+    is_negative_changed = diff / avg <= eps and value <= ref
+    assert is_negative_changed, f"Relative difference exceeded {eps} threshold. Absolute difference: {diff}"
```

Comment:
```suggestion
    is_positive_changed = value > ref or diff / avg <= eps
    assert is_positive_changed, f"Relative difference exceeded {eps} threshold. Absolute difference: {diff}"
```
Isn't this right?

Reply:
Thanks for the interest even though it's still draft, it will be handled at the another PR